src/components/sidebar/sidebar.scss

```diff
@@ -3,7 +3,7 @@
   top: 0;
   bottom: 0;
   left: 0;
-  min-height: 93.5vh;
+  min-height: 92.5vh;
```

Comment:
1. need to fix the hight issue the height of the navbar looks in appropriate.
![image](https://user-images.githubusercontent.com/32647423/86256114-8b6fa780-bbd5-11ea-9869-6a644294d678.png)

2. Navbar content overlaps below 1050px. Responsiveness needs to be taken into handle.
![image](https://user-images.githubusercontent.com/32647423/86255966-611dea00-bbd5-11ea-954c-26b4639fc223.png)

Reply:
Cool will work on that!